# defain environment sensor sht30 message format
# Header
Header header
# temperature
float32 temperature
# humidity
float32 humidity
